playbooks/openshift/post-provision-openstack.yml

```diff
@@ -18,7 +18,6 @@
 # provision cinder volume
 - import_playbook: cinder-registry.yml
   when:
-  - openshift_hosted_registry_storage_kind is defined
```

Comment:
what you had here worked just fine - not sure why the change? I'd say keep it as-is and just key off the `openshift_hosted_registry_storage_kind` and `openshift_hosted_registry_storage_openstack_volumeID` - i.e.: if the latter is NOT set, we'll provision one to be used ... again, that's the way you had it and should work fine as-is